ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -194,7 +194,19 @@ module.exports = Marionette.LayoutView.extend({
     }
   },
   validate() {
-    return this.querySettings.currentView.validate()
+    const errorMessages = this.querySettings.currentView.validate()
+    const filters = this.queryAdvanced.currentView.getFilters().filters
+    for (let i = 0; i < filters.length; i++) {
```

Comment:
Would recommend refactoring the two validate methods to use a shared method

Reply:
It could be added in the validation util file cassandra added https://github.com/codice/ddf/pull/5594/files#diff-74e3a298e21c93687fcc7ab28492b1f6R19

Reply:
👍